README.md

```diff
@@ -23,5 +23,10 @@ Write parameters like this.
 var geometry = new Point(-122.129797, 47.640049) { SRID = 4326 };
 var geometryWriter = new SqlServerBytesWriter { IsGeography = true };
 var bytes = geometryWriter.Write(geometry);
-command.Parameters.AddWithValue(parameterName, new SqlBytes(bytes));
+var parameter = command.Parameters
+    .AddWithValue(parameterName, new SqlBytes(bytes));
+
+// TODO: Set these if you're using Microsoft.Data.SqlClient
```

Comment:
Sometimes it works on System.Data.SqlClient 4.7.0+, but only for certain target frameworks and operating systems. I figured I should keep it simple and just say it works on [the new Microsoft.Data.SqlClient](https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/) library.